tasks/go.py

```diff
@@ -391,6 +399,34 @@ def is_excluded(pkg):
                         print("Skipping {} ({}) excluded in .wwhrd.yml".format(package, license))
                     else:
                         licenses.append("core,\"{}\",{}".format(package, license))
+
+    # Additional Licenses
+    for pkg, lic in additional.items():
+        url = urlparse(lic)
+        url = url._replace(scheme='https', netloc=url.path, path='')
+        try:
+            resp = requests.get(url.geturl())
+            resp.raise_for_status()
+
+            with tempfile.TemporaryDirectory() as tempdir:
+                with open(os.path.join(tempdir, 'LICENSE'), 'w') as lfp:
+                    lfp.write(resp.text)
+                    lfp.flush()
+
+                    temp_path = os.path.dirname(lfp.name)
+                    result = ctx.run("go run github.com/go-enry/go-license-detector/v4/cmd/license-detector -f json {}".format(temp_path))
+                    if result.stdout:
+                        results = json.loads(result.stdout)
+                        for project in results:
+                            if 'error' in project:
+                                continue
+
+                            # we get the first match
+                            license = project['matches'][0]['license']
+                            licenses.append("core,\"{}\",{}".format(pkg, license))
+        except RequestException:
+            continue
```

Comment:
I think we should not catch this with the new implementation: we want to know that a license has gone missing from the Internet. If you think we should catch it, we should at least print something for debuggability

Reply:
Makes sense, I think we might still want to catch it and continue, but as you state print an error for debuggability. I think we should catch it because the section is prone to bad user input. We can always change it in the future.